multihost_test/bz_automation/test_automation.py

```diff
@@ -127,3 +127,31 @@ def test_2091062(self, multihost, create_localusers):
         client.logout()
         assert "pam_motd: error scanning directory" not in \
                execute_cmd(multihost, "cat /var/log/secure").stdout_text
+    def test_pam_faillock_audit(self, multihost, create_localusers, bkp_pam_config):
+        """
+        :title: Pam_faillock audit events duplicate uid.
+        :id: c62fda84-401b-11ee-90bd-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2231556
+        :setup:
+            1. Enable pam_faillock in the PAM stack (either with authselect "authselect
+                enable-feature with-faillock" or manually)
```

Comment:
Since you are using `authselect` in the code I think you can remove `(either with authselect "authselect enable-feature with-faillock" or manually)`